### PR TITLE
porting code from ovirt-ansible

### PR DESCRIPTION
**Pull request by maorlipchuk**

### CODE REVIEW

**Comment by mwperina** [comment]
Generally it looks good to me, but I'm missing some important files for Galaxy (LICENSE, meta, ...) and CI (automation, RPM packaging, ...)

File: automation/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.tox
```
Comment (machacekondra) [inline automation/.gitignore]: .gitignore file should be in root directory.

File: automation/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (machacekondra) [inline automation/.travis.yml]: please remove this file for now

File: automation/test-requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ansible-lint
```
Comment (machacekondra) [inline automation/test-requirements.txt]: please remove this file , we don't run any tests now.

File: examples/roles [hunk 4]
```diff
@@ -0,0 +1 @@
+../roles
```
Comment (machacekondra) [inline examples/roles]: you can remove this file

File: meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  author: "Maor Lipchuk"
+  description: "oVirt Ansible roles"
```
Comment (machacekondra) [inline meta/main.yml]: description: "Disaster recovery for oVirt engine."
or anything what describes the actual role

File: meta/main.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  company: Red Hat, Inc.
+  license: Apache License 2.0
+  min_ansible_version: 2.4
+  platforms:
+    - name: EL
+      versions:
+        - all
```
Comment (machacekondra) [inline meta/main.yml]: EL just 7

File: meta/main.yml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    - name: Fedora
+      versions:
+      - 24
```
Comment (machacekondra) [inline meta/main.yml]: here version all

File: automation/tox.ini [hunk 8]
```diff
@@ -0,0 +1,35 @@
+[tox]
```
Comment (machacekondra) [inline automation/tox.ini]: Please remove this file, I will add one common tox file for all ansible roles

File: automation/.gitignore (hunk 1 above)
Comment (maorlipchuk) [reply automation/.gitignore]: done

File: automation/.travis.yml (hunk 2 above)
Comment (maorlipchuk) [reply automation/.travis.yml]: done

File: automation/test-requirements.txt (hunk 3 above)
Comment (maorlipchuk) [reply automation/test-requirements.txt]: done

File: automation/tox.ini (hunk 8 above)
Comment (maorlipchuk) [reply automation/tox.ini]: done

File: meta/main.yml (hunk 5 above)
Comment (maorlipchuk) [reply meta/main.yml]: Done

File: meta/main.yml (hunk 6 above)
Comment (maorlipchuk) [reply meta/main.yml]: done

File: meta/main.yml (hunk 7 above)
Comment (maorlipchuk) [reply meta/main.yml]: done

File: examples/roles (hunk 4 above)
Comment (maorlipchuk) [reply examples/roles]: done

**Comment by machacekondra** [comment]
Looks good to me, I think we can do an alpha release. @mwperina

**Pull request merged**